src/index/coinstatsindex.cpp

```diff
@@ -123,7 +123,7 @@ bool CoinStatsIndex::CustomAppend(const interfaces::BlockInfo& block)
         // pindex variable gives indexing code access to node internals. It
         // will be removed in upcoming commit
         const CBlockIndex* pindex = WITH_LOCK(cs_main, return m_chainstate->m_blockman.LookupBlockIndex(block.hash));
-        if (!m_chainstate->m_blockman.UndoReadFromDisk(block_undo, *pindex)) {
+        if (!m_chainstate->m_blockman.UndoRead(block_undo, *pindex)) {
```

Comment:
In commit 0dd386994bafd67081c4521673018ddd22e2d63c

Can we call this `ReadBlockUndo` instead?

Reply:
That was inconsistent, indeed - [fixed](https://github.com/bitcoin/bitcoin/compare/0dd386994bafd67081c4521673018ddd22e2d63c..2ff0ea366c61b2fcb80ad1f711480915c7a7aa2e)!